backend/services/grading_vector_store.py

```
@@ -0,0 +1,161 @@
+from typing import List, Dict, Any, Optional
+import asyncio
+from langchain.tools import BaseTool
+from langchain.schema import BaseMessage
+
+from services.grading_embedding_service import grading_embedding_service
+from utilities.debug_logging import debug_log
+
+class GradingVectorStore:
+    """Vector store service for retrieving grading materials context"""
+    
+    def __init__(self):
```

Ruff (0.14.1)

Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)